amp/WEB-INF/src/org/digijava/module/aim/util/AuditLoggerUtil.java

```diff
@@ -387,7 +387,7 @@ public static Collection getLogByPeriod(int interval) {
     public static void deleteLogsByPeriod(String interval) {
         try {
             String qryStr = "delete from " + AmpAuditLogger.class.getName()
-                    + " where action<>'login' and (loggedDate <= :dateParam or loggedDate=null)";
```

Comment:
The AuditCleanerJob remove older logs except for the login ones, therefore there is no need to change the code here.